sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala

```diff
@@ -120,6 +120,7 @@ object JDBCRDD extends Logging {
       case StringStartsWith(attr, value) => s"${attr} LIKE '${value}%'"
       case StringEndsWith(attr, value) => s"${attr} LIKE '%${value}'"
       case StringContains(attr, value) => s"${attr} LIKE '%${value}%'"
+      case In(attr, value) if value.isEmpty => s"IF(${attr} IS NULL, NULL, false)"
```

Comment:
`IF` is not a part of the SQL standard, use: `CASE WHEN $attr IS NULL THEN NULL ELSE FALSE END`